lib/components/note-preview/index.tsx

```diff
@@ -106,6 +142,8 @@ export const NotePreview: FunctionComponent<Props> = ({
 
 const mapStateToProps: S.MapState<StateProps, OwnProps> = (state, props) => ({
   fontSize: state.settings.fontSize,
+  isDialogOpen: state.ui.dialogs.length > 0,
+  isNoteInfoOpen: state.ui.showNoteInfo,
```

Comment:
might also consider reframing this from the perspective of the inner component and the API we want to expose from it…

```ts
copyFullRenderedContent: state.ui.dialogs.length = 0 && !state.ui.showNoteInfo
```

…this way it's clearer why we're using them and easier to push into a known testable state.